src/System.Private.ServiceModel/tools/IISHostedWcfService/Web.config

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <!--Copyright (c) Microsoft Corporation.  All Rights Reserved.-->
+<configuration>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel" switchValue="Information, ActivityTracing" propagateActivity="true">
+        <listeners>
+          <add name="traceListener" type="System.Diagnostics.XmlWriterTraceListener" initializeData="c:\log\Traces.svclog" />
```

Comment:
In the case of CI, there are multiple repos. It will be hard to tell which trace belongs to which service when trace files are only differentiated by prefixed GUIDs. I was wondering if something like this will work: `initialzeData="..\..\..\bin\WcfServicelog\traces.svclog"`, which is supposed to point to `repo-root\bin` and `bin` folder is one of git ignored folders.

If above does not work (or you don't like it for any reason), can you please just update the hard coded path to `c:\WCFTest\log\traces.svclog`? Because there is another hard coded path `c:\WCFTest\test.crl` in `Util.cs`, putting all external resources/dependencies in one root folder (`c:\WCFTest`) will make setup easier.

> Shall I change the default value from Information to Warning to avoid the trace file growing too fast?

Warning may be the right level of logging as most of the time we care only when there are problems, but I don't feel strongly in any case.